README.md

```diff
@@ -254,6 +255,8 @@ fastify.get('/external', { /* Hooks can be used here */ }, async (req, reply) =>
 });
 ```
 
+- `revokeToken(Token, tokenType, params, callback)`: A function that use for remove the current access_token or refresh_token. If the callback is not passed it will return a promise. The callback call or promise resolution returns an `void`
```

Comment:
```suggestion
- `revokeToken(Token, tokenType, params, callback)`: A function to revoke the current access_token or refresh_token on server side. If the callback is not passed it will return a promise. The callback call or promise resolution returns `void`
```